spec/lib/newspaper_works/ingest/image_ingest_issues_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+RSpec.describe NewspaperWorks::Ingest::ImageIngestIssues do
+  include_context 'ingest test fixtures'
+
+  let(:lccn) { 'sn93059126' }
+
+  let(:publication) { NewspaperWorks::Ingest::PublicationInfo.new(lccn) }
+
+  let(:pub_path) { File.join(tiff_fixtures, lccn) }
+
+  let(:expected_paths) do
+    entries = Dir.entries(pub_path).map { |p| File.join(pub_path, p) }
+    entries.select { |p| File.directory?(p) && !File.basename(p).start_with?('.') }
+  end
+
+  describe " construction and metadata" do
+    it "constructs with path and publication" do
+      issues = described_class.new(pub_path, publication)
```

Comment:
This line is repeated in every spec, consider DRYing

Reply:
Agreed, will move this to a `let`.